test/unit/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -113,10 +141,12 @@ describe('CaseTypeTab', () => {
       'marriageCertificate',
       'coRespondent',
       // 'SolicitorCoRespondent', - TODO - uncomment this when we go live with AOS pack 2
+      // 'serviceApplication',
```

Comment:
why?

Reply:
Sorry, typo, forgot to uncomment. Thanks

Reply:
PS: @yarpo. The // 'SolicitorCoRespondent' comment was already there and i have left it as is.